Lib/concurrent/futures/_base.py

```diff
@@ -558,17 +558,26 @@ def set_exception(self, exception):
 class Executor(object):
     """This is an abstract base class for concrete asynchronous executors."""
 
-    def submit(self, fn, /, *args, **kwargs):
-        """Submits a callable to be executed with the given arguments.
+    def submit_with_name(self, fn, name=None, /, *args, **kwargs):
+        """Submits a callable to be executed with the given arguments and also with a name.
 
         Schedules the callable to be executed as fn(*args, **kwargs) and returns
         a Future instance representing the execution of the callable.
 
         Returns:
             A Future representing the given call.
         """
+
         raise NotImplementedError()
```

Comment:
@mdickinson `submit_with_name` was created in `_base.Executor` which was not Implemented so the same is true for `ProcessPoolExecutor`